### PR TITLE
DHFPROD-5826: Array of booleans and numbers not being handled in Mapp…

### PR DESCRIPTION
**Pull request by briantang**
…ing source data

### Description

- Update to Cust1.json in reference-entity-model project to accommodate edge cases
- Fixed issue with rendering of source data for cases where value is an array of booleans or array of numbers.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: examples/reference-entity-model/input/json/Cust1.json [hunk 1]
```diff
@@ -40,5 +40,22 @@
   "PIN": 2172,
   "Created": "04/03/2017",
   "Status": "New Customer",
-  "SSN": "111-11-1111"
+  "SSN": "111-11-1111",
+  "numbersArray" : [
```
Comment (srinathgit) [inline examples/reference-entity-model/input/json/Cust1.json:44]: I think it would be better to have UI tests for these values rather than adding it to reference project input data. These new data don't have relevance to "Customer" data

Comment (briantang) [reply examples/reference-entity-model/input/json/Cust1.json:44]: Thanks, @srinathgit  that's fair. I initially checked with @bsrikan beforehand so I'm curious what he thinks? I could also make it a new input file unrelated to "Customer" data.

Comment (srinathgit) [reply examples/reference-entity-model/input/json/Cust1.json:44]: I think we would like to keep reference project as clean as possible. @bsrikan , let us know your thoughts

Comment (bsrikan) [reply examples/reference-entity-model/input/json/Cust1.json:44]: When we were reviewing the test plan we wanted any enhanced test data to be added to reference project, because it is pushed to HC nightly and most of our engineers and PM use it for reviewing/testing PR.

Comment (bsrikan) [reply examples/reference-entity-model/input/json/Cust1.json:44]: As for tests, Brian has added unit tests for the fix.

Comment (wooldridge) [reply examples/reference-entity-model/input/json/Cust1.json:44]: I agree with @srinathgit, the example reference project should be more like a demo with (Customer) data that ties together (although it is also intended to cover many data use cases).

Comment (briantang) [reply examples/reference-entity-model/input/json/Cust1.json:44]: Looks like we're leaning towards taking out this enhancement to test data with certain edge cases to keep the 'Customer' data clean. @rjrudin could you confirm? And I can go ahead and take that out of the PR.

Comment (wooldridge) [reply examples/reference-entity-model/input/json/Cust1.json:44]: @briantang you did add more test data to common.data.ts to cover these cases and I think that's fine. Let's pull those examples out of the reference project for now and push this PR through. I can't see @rjrudin having a problem with this. :-)

Comment (briantang) [reply examples/reference-entity-model/input/json/Cust1.json]: Thanks @wooldridge, will leave it out for now so we can get this bug fix through.

Comment (rjrudin) [reply examples/reference-entity-model/input/json/Cust1.json]: The key to me is that the entity model isn't being used to determine how data in the source panel is displayed - that's not the case, right?

Comment (wooldridge) [reply examples/reference-entity-model/input/json/Cust1.json]: @rjrudin that is not the case, display of source data is based on the source data.
